controller/space_iterations_blackbox_test.go

```diff
@@ -0,0 +1,558 @@
+package controller_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/fabric8-services/fabric8-wit/account"
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/fabric8-services/fabric8-wit/app/test"
+	"github.com/fabric8-services/fabric8-wit/application"
+	. "github.com/fabric8-services/fabric8-wit/controller"
+	"github.com/fabric8-services/fabric8-wit/gormapplication"
+	"github.com/fabric8-services/fabric8-wit/gormsupport"
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/iteration"
+	"github.com/fabric8-services/fabric8-wit/ptr"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	"github.com/fabric8-services/fabric8-wit/space"
+	"github.com/fabric8-services/fabric8-wit/spacetemplate"
+	testsupport "github.com/fabric8-services/fabric8-wit/test"
+	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+	"github.com/goadesign/goa"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type SpaceIterationControllerTestSuite struct {
+	gormtestsupport.DBTestSuite
+	db           *gormapplication.GormDB
+	testIdentity account.Identity
+	testDir      string
+}
+
+func TestSpaceIterationController(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &SpaceIterationControllerTestSuite{DBTestSuite: gormtestsupport.NewDBTestSuite("../config.yaml")})
+}
+
+func (s *SpaceIterationControllerTestSuite) SetupTest() {
+	s.DBTestSuite.SetupTest()
+	s.db = gormapplication.NewGormDB(s.DB)
+	testIdentity, err := testsupport.CreateTestIdentity(s.DB, "SpaceIterationControllerTestSuite user", "test provider")
+	require.NoError(s.T(), err)
+	s.testIdentity = *testIdentity
```

Comment:
Do you really need this test identity? I mean we have identities created for you when you use test fixtures.